### ISSUE TITLE
Technical names toggle does not clear field value names

### ISSUE THREAD

**Issue by johndahl-now** [arnoudkooi/ServiceNow-Utils#289]
When using the form white-space-double-click action to toggle the Technical Name display feature, the action does not remove the technical names from the choice list field values.

Build name: Tokyo
Build date: 07-26-2022_2149
Build tag: glide-tokyo-07-08-2022__patch0-07-19-2022
UI: Next Experience UI

Toggled On:
<img width="799" alt="toggled_on" src="https://user-images.githubusercontent.com/103326903/185790301-523d9b8d-a64d-40ae-8d78-4092f63b3982.png">

Toggled Off:
<img width="690" alt="toggled_off" src="https://user-images.githubusercontent.com/103326903/185790306-1da52e8e-bf33-49b1-872c-472da91bae84.png">

**Comment by arnoudkooi** [comment]
Correct, this is since not been implemented,  I did make a mention of it when announced on Twitter.
I'll move this one to a discussion/idea and will have a look at it

[https://twitter.com/sn_utils/status/1483887850521255939](<img width="610" alt="image" src="https://user-images.githubusercontent.com/840781/185791385-201c9165-ff23-4326-958b-646b78abc9ce.png">)
